### ISSUE TITLE
Accessibility: HighContrast: When RenderMode is System for toolStrip/statusStrip, the text of toolStripButton/SplitButton/DropDownButton is not visible after user mouse hover or keyboard focus on HC black/white

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#5502]
* .NET Core Version: 
.Net SDK 6.0.100-rc.1.21417.19

* Have you experienced this same bug with .NET Framework?: 
Yes

* More Info
This issue cannot repro when setting ToolStrip/StatusStrip's RenderMode property to Professional or ManagerRenderMode

**Minimal repro:**
1. Turn on Windows high contrast, and choose a theme: High Contrast Black/White
2. Open attached application in VS
[WinFormsApp54.zip](https://github.com/dotnet/winforms/files/7029758/WinFormsApp54.zip)
3. Build and run app
4. Mouse over or press Left/Right keyboard arrow to focus any of toolStripItems

**Expected behavior:**
The text of toolStripButton/SplitButton/DropDownButton is visible after user mouse hover or keyboard focus on HC black/white

**Actual behavior:**
The text of toolStripButton/SplitButton/DropDownButton is not visible after user mouse hover or keyboard focus on HC black/white
![image](https://user-images.githubusercontent.com/26474449/130404801-921371b4-2ead-4ebb-934f-43c60588cd22.png)

**Comment by ArtemTatarinov** [comment]
> I don't know what the specific cause was of our original implementation. What does it look like in HC and non HC themes when you set both flags to true? We might need to check it out on both win10 and win11 come to think of it. It seems logical that we need to have the flags set so that we can actually see anything!

Those flags don't affect non HC theme, only the HC one. In HC theme `true` flag with the certain additional modifications at the `ToolStripSystemRenderer` class fix the issue: selected toolstrip buttons will have the filled background. The issue is also reproduced on win11 with HC themes.

**Comment by Tanya-Solyanik** [comment]
> The issue is also reproduced on win11 with HC themes.
Good job testing on Win11!

**Comment by Olina-Zhang** [comment]
Verified this issue in the latest .net SDK 6.0 RC2 build: 6.0.100-rc.2.21460.37, it is fixed in HC black and white themes for toolStripButton/DropDownButton/SplitButton
![image](https://user-images.githubusercontent.com/26474449/133061484-4cb32ecb-d7da-41a9-8504-d79d780c23ae.png)
But hit a known issue: https://github.com/dotnet/winforms/issues/5676 when checking and focusing items:
![image](https://user-images.githubusercontent.com/26474449/133061772-53cdcf69-9531-4cab-930d-44c3a78a68a7.png)

**Comment by Ashley-Li** [comment]
Verified this issue with .NET SDK 7.0.100-alpha.1.21479.26, it is fixed in HC black and white themes for toolStripButton/SplitButton/DropDownButton.
![image](https://user-images.githubusercontent.com/56664234/135401637-b681324d-0ba9-4d95-a994-0ace52381d4b.png)
